p/themes/base-theme/template.css

```diff
@@ -1066,18 +1071,19 @@ input[type="search"] {
 	position: static;
 }
 
-.flux .item {
-	line-height: 40px;
+.flux .flux_header .item {
+	padding: 0.5rem 0;
+	line-height: 1.5rem;
 	white-space: nowrap;
 }
 
-.flux .item.manage,
-.flux .item.link {
+.flux .flux_header .item.manage,
+.flux .flux_header .item.link {
 	width: 40px;
 	text-align: center;
 }
 
-.flux .item.website {
+.flux .flux_header .item.website {
```

Comment:
As an aside, the PR currently has this visual regression:
<img src=https://user-images.githubusercontent.com/202757/188716141-f0707d75-5c0e-4daf-ba18-f1b597671213.png width=40%>

Reply:
thanks! fixed